ajax/libs/video.js/package.json

```diff
@@ -13,7 +13,7 @@
     {
       "basePath": "dist",
       "files": [
-        "**/*.+(js|css|map|woff|ttf|svg|eot|swf)"
+        "**/!(*cjs|*es|*amd).+(js|css|map|woff|ttf|svg|eot|swf)"
```

Comment:
Are you worried about the files related to amd in the future? Ask this because I didn't see any files containing `amd` currently.